src/test/ParkVehicleTest.java

```diff
@@ -0,0 +1,30 @@
+package test;
+
+import main.models.terminal.Terminal;
+import main.models.terminal.TerminalWorker;
+import main.models.vehicles.*;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParkVehicleTest {
+
+    @Test
+    public void testParkingVehicle(){
+        Terminal smallTrainTerminal = new Terminal.TerminalBuilder(TrainSize.SMALL)
```

Comment:
There could definitely be more tests. Also, if you want to have this exact object available in every test, you would define it as class attribute and initialize it in `@BeforeAll`. Take a look at [this](https://www.baeldung.com/junit-before-beforeclass-beforeeach-beforeall) as well.

Reply:
Added new test, and beforeAll initalizer